src/components/EditCoursePage/index.jsx

```diff
@@ -137,6 +142,49 @@ class EditCoursePage extends React.Component {
       video: { src: courseData.videoSrc },
     };
   }
+  showModal(submitCourseData) {
+    const {
+      targetRun,
+    } = this.state;
+
+    if (targetRun && !(targetRun.status === PUBLISHED)) {
+      // Submitting Run for review, show modal, and temporarily store form data until
+      // we have a response for how to continue.
+      this.setState({
+        submitCourseData,
```

Comment:
This whole temporarily storing data in state might come in handy with the pristine work we have coming up